### ISSUE TITLE
Apparent implicit assumption about electron spectrum

### ISSUE THREAD

**Issue by nj-vs-vh** [carmeloevoli/heath-2324#2]
https://github.com/carmeloevoli/heath-2324/blob/6014908d2e46d1c57691a9b648b8afe358b9cb83/sections/leptonic.tex#L393

I'm going over the calculations and try to repeat them, primarily to understand them better myself. I hope it's ok to ask for clarifications in the form of github issues!

In particular, I am able to reproduce this result only by assuming p > 3 -- then a large gamma_max/gamma_min ratio is taken to some negative power (both in the numerator and the denominator), and we can ignore this term, arriving at 1/gamma_min for the ratio of integrals.

Is this true, or did I make a mistake?

**Comment by nj-vs-vh** [comment]
In PDF, the relevant equation is at the top of page 12

**Comment by carmeloevoli** [comment]
> In particular, I am able to reproduce this result only by assuming p > 3 -- then a large gamma_max/gamma_min ratio is taken to some negative power (both in the numerator and the denominator), and we can ignore this term, arriving at 1/gamma_min for the ratio of integrals.
> 
> Is this true, or did I make a mistake?

Yes, that is true. It is a common assumption as electrons are injected by something steeper than 2, and, due to energy losses, additionally softened by another factor 1. You are right, better to note it explicitly in the text! 👍
